app/views/Settings.js

```diff
@@ -81,11 +122,11 @@ export const SettingsScreen = ({ navigation }) => {
         <Section>
           <Item
             label={
-              isLogging
+              isLogging && isBGLocationLogging
```

Comment:
these two variables seem to track the same thing. So they could do with different names.

perhaps: `isUserParticipating && isBgLocationLogging`